src/Website/Services/SiteNews/InMemorySiteNewsProvider.cs

```diff
@@ -0,0 +1,50 @@
+﻿// <copyright file="InMemorySiteNewsProvider.cs" company="Clicker Heroes Tracker">
+// Copyright (c) Clicker Heroes Tracker. All rights reserved.
+// </copyright>
+
+namespace Website.Services.SiteNews
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+
+    /// <summary>
+    /// In memory provider for site news.
+    /// This is used when there is no azure table storage setup.
+    /// </summary>
+    public class InMemorySiteNewsProvider : ISiteNewsProvider
+    {
+        private readonly IDictionary<DateTime, IList<string>> siteNewsDictionary = new Dictionary<DateTime, IList<string>>();
```

Comment:
Consider locking (I don't think a ConcurrentDictionary would work since there's the list inside which also may mutate)? Not that we really expect this impl to be used in prod. Maybe at least call out in the comment on the class that it shoudln't be used in prod because of thread-safety.